src/components/CodeView/index.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+import styles from './styles.module.scss';
+import { getLanguage, getLines, mapWithDepth } from './utils';
+import refractor from '../../refractor';
+
+type PublicProps = {
+  mimeType: string;
+  content: string;
+};
+
+// This function mimics what https://github.com/rexxars/react-refractor does,
+// but we need a different layout to inline comments so we cannot use this
```

Comment:
Why couldn't you use [markers](https://github.com/rexxars/react-refractor/#markers)?

The tooltip example seemed like it would allow us to do it, no?

```jsx
<Refractor
  language="js"
  value={source}
  markers={[
    {line: 1, className: 'no-not-use-foo-in-examples'},
    {line: 3, component: props => (
      <TooltipedLine tooltipText="Prefer template for string concatenation">
        {props.children}
      </TooltipedLine>
    )}
  ]}
/>
```

Reply:
We can't have markers and line numbers.